projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -83,6 +85,9 @@ public function get_item_permissions_check( $request ) { // phpcs:ignore Generic
 	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
 	 */
 	public function get_item( $request ) { // phpcs:ignore Generic.CodeAnalysis.UnusedFunctionParameter, VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
+		if ( ! ( new Host() )->is_wpcom_platform() && get_option( 'wpcom_admin_interface' ) !== 'wp-admin' ) {
```

Comment:
As noted in the other PR, we don't need the `wpcom_admin_interface` option check.
```suggestion
		if ( ! ( new Host() )->is_wpcom_platform() ) {
```

Reply:
I committed your suggestion, thanks! Can you confirm this will not affect AT sites with `wpcom_admin_interface` set? Just being super cautious here because this applies to AT too :)

Reply:
Actually you are right since `is_wpcom_platform` [returns](https://github.com/Automattic/jetpack/blob/4e25fd02631780b4da5544280869ee154b95ba7e/projects/packages/status/src/class-host.php#L77) `true` for WoA sites. All good here, thanks!